assets/app/scripts/app.js

```diff
@@ -191,6 +191,17 @@ angular
         return durationFilter($(this).attr("data-timestamp")) || existing;
       });      
     }, 1000);
+  })
+  .run(function($rootScope){
```

Comment:
this whole change was taken out in the latest version of my branch, guessing it got added back in during your rebase